### PR TITLE
OTS-283: docs refactor

### PR DESCRIPTION
**Pull request by jneimantokbox**
Docs refactor per request from engineering.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/opentok/one-to-one-sample-apps/pull/145" target="_blank">ReviewNinja

**Comment by jneimantokbox** [comment]
@adrice727 , @Lucashuang0802 , @marinaserranomontes : Please review and merge if approved.

File: one-to-one-sample-app/README.md [hunk 1]
```diff
@@ -15,35 +15,36 @@ With this sample app, you can:
...
 
 For example, suppose one user is using a web (JS) version of the One-to-One Communication Sample App and another user is using a mobile version (Android or iOS). If they are both using the same **Session ID** and **API Key**, they can subscribe to each other’s audio and video streams, and the user interface rendered on both devices will allow them to interact with each other and take advantage of all the features of the sample app.
 
 Use the following approach to try this out:
 
 1. Configure a web and mobile user with the required  **Session ID**, **Token**, and **API Key** values, using the same **Session ID** and **API Key** for each.
 
-2. For the mobile users, ensure that `SUBSCRIBE_TO_SELF = false` (Android) and `subscribeToSelf = NO` (iOS) in the configuration.
+2. For mobile users, ensure that `SUBSCRIBE_TO_SELF = false` (Android) and `subscribeToSelf = NO` (iOS) in the configuration.
```
Comment (Lucashuang0802) [inline one-to-one-sample-app/README.md]: We do have this section anymore. So, let's remove it. 😃

**Comment by Lucashuang0802** [comment]
And I think we can merge the README at the root and https://github.com/opentok/one-to-one-sample-apps/tree/master/one-to-one-sample-app. It's sort of duplicate. @jneimantokbox

File: one-to-one-sample-app/README.md (hunk 1 above)
Comment (marinaserranomontes) [inline one-to-one-sample-app/README.md]: @Lucashuang0802 what do you mean?

Comment (Lucashuang0802) [inline one-to-one-sample-app/README.md]: `subscribeToSelf` is removed since first release.

**Comment by Lucashuang0802** [comment]
👍

File: one-to-one-sample-app/README.md (hunk 1 above)
Comment (marinaserranomontes) [inline one-to-one-sample-app/README.md]: In Android we continue having the "SUBSCRIBETOSELF" parameter

**Pull request merged**